ios/engine/KMEI/KeymanEngine/Classes/Resource Management/ResourceDownloadManager.swift

```diff
@@ -530,12 +553,27 @@ public class ResourceDownloadManager {
     return { package, error in
       if let package = package {
         // Do not send notifications for individual resource updates.
-//        let resourceIDs: [Resource.FullID] = resources.map { $0.typedFullID }
-//        do {
-//          try ResourceFileManager.shared.install(resourcesWithIDs: resourceIDs, from: package)
-//        } catch {
-//          log.error("Error updating resources from package \(package.id)")
-//        }
+        let resourceIDs: [Resource.FullID] = resources.map { $0.typedFullID }
+        do {
+          if let keyboards = resources as? [InstallableKeyboard], let package = package as? KeyboardKeymanPackage {
+            // TEMP:  currently required because downloaded keyboards aren't actually in packages.
+            keyboards.forEach { keyboard in
+              if let updatedKbd = package.findResource(withID: keyboard.typedFullID) {
+                Manager.shared.updateUserKeyboards(with: updatedKbd)
+
+                if Manager.shared.currentKeyboard?.fullID == keyboard.fullID {
+                  // Issue:  does not actually trigger a reload if the user isn't within the Settings view hierarchy
+                  // Fixing this requires a refactor of `shouldReloadKeyboard`.
```

Comment:
This should be done within its own PR.  I already know what needs to be done and the approximate scale involved, but I believe it should be triaged until later in the sprint; we have more urgent things to address.